nerdlets/network-telemetry-overview/network-telemetry-nerdlet.jsx

```diff
@@ -40,7 +40,7 @@ export default class NetworkTelemetryNerdlet extends React.Component {
     return (
       <div className='select-account'>
         <HeadingText type={HeadingText.TYPE.HEADING_1}>
```

Comment:
Since we're reducing the icon size, how about making this: 
```js
 <HeadingText type={HeadingText.TYPE.HEADING_2}>
```

![image](https://user-images.githubusercontent.com/20293876/106334296-32d1bd00-623f-11eb-8055-2911c66da903.png)